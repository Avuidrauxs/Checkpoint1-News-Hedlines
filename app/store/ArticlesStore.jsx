import { EventEmitter } from 'events';
import Dispatcher from '../dispatcher';

/**
 *Listens and stores data from the action according to their action type
 *
 * @class ArticleStore
 * @extends {EventEmitter}
 */
class ArticlesStore extends EventEmitter {
  /**
   * ArticleStore constructor
   * @return {null} - returns no value
   */
  constructor() {
    super();
    this.articles = [];
    this.articleTitle = '';
    this.getAllNewsArticles = this.getAllNewsArticles.bind(this);
  }

/**
 * this method just sets ths articles store with articles
 * @memberof ArticlesStore
 * @return {array} - returns an array of articles
 */
  getAllNewsArticles() {
    return { articles: this.articles, articleSource: this.articleTitle };
  }
  /**
   * This function listens for payLoad from the action and stores them
   * according to their action type.
   * @param {object} action
   *@return {null} - returns no value
   * @memberof ArticlesStore
   */
  handleActions(action) {
    switch (action.type) {
      case 'FETCH_ALL_ARTICLES':
        this.articles = action.articles.articleList;
        this.articleTitle = action.articles.articleSource;
        this.emit('change');
        break;
      default:
    }
  }

}

const articlesStore = new ArticlesStore();
Dispatcher.register(articlesStore.handleActions.bind(articlesStore));

export default articlesStore;
