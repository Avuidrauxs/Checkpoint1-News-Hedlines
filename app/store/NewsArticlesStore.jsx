import { EventEmitter } from 'events';
import Dispatcher from '../dispatcher';

/**
 *Listens and stores data from the action according to their action type
 *
 * @class NewsArticleStore
 * @extends {EventEmitter}
 */
class NewsArticlesStore extends EventEmitter {
  /**
   * ArticleStore constructor
   * @return {null} - returns no value
   */
  constructor() {
    super();
    this.articles = [];
    this.articleSource = '';
    this.getAllNewsArticles = this.getAllNewsArticles.bind(this);
  }

/**
 * this method just sets ths articles store with articles
 * @memberof ArticlesStore
 * @return {array} - returns an array of articles
 */
  getAllNewsArticles() {
    return { articles: this.articles, articleSource: this.articleSource };
  }
  /**
   * This function listens for payLoad from the action and stores them
   * according to their action type.
   * @param {object} action
   *@return {null} - returns no value
   * @memberof ArticlesStore
   */
  handleActions(action) {
    switch (action.type) {
      case 'FETCH_ALL_ARTICLES':
        this.articles = action.articles.articleList;
        this.articleSource = action.articles.articleSource;
        this.emit('change');
        break;
      default:
    }
  }

}

const articlesStore = new NewsArticlesStore();
Dispatcher.register(articlesStore.handleActions.bind(articlesStore));

export default articlesStore;
