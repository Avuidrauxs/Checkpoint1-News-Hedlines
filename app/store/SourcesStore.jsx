import { EventEmitter } from 'events';
import Dispatcher from '../dispatcher';

/**
 *Listens and stores data from the action according to their action type
 *
 * @class SourcesStore
 * @extends {EventEmitter}
 */
class SourcesStore extends EventEmitter {
  /**
   * SourcesStore constructor
   * @return {null} - returns no value
   */
  constructor() {
    super();
    this.sources = [];
    this.getAllNewsSources = this.getAllNewsSources.bind(this);
  }
  /**
   * this method just sets ths sources store with sources
   * @memberof SourcesStore
   * @return {array} - returns an array of news sources
   */
  getAllNewsSources() {
    return this.sources;
  }
  /**
   * This function listens for payLoad from the action and stores them
   * according to their action type.
   * @param {object} action
   *@return {null} - returns no value
   * @memberof SourcesStore
   */
  handleActions(action) {
    switch (action.type) {
      case 'FETCH_ALL_NEWS':
        this.sources = action.sources;
        this.emit('change');
        break;
      default:
    }
  }

}

const sourcesStore = new SourcesStore();
Dispatcher.register(sourcesStore.handleActions.bind(sourcesStore));

export default sourcesStore;
