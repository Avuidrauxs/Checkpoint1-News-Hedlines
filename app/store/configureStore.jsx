import { createStore, compose, combineReducers, applyMiddleware } from 'redux';
import { searchTextReducer, getAllNewsReducer, fetchArticlesReducer, googleSignInReducer } from 'reducers';

const thunk = require('redux-thunk').default;

const configureStore = (initialState = {}) => {
  const reducer = combineReducers({
    searchText: searchTextReducer,
    getAllNews: getAllNewsReducer,
    fetchArticles: fetchArticlesReducer,
    googleSignIn: googleSignInReducer,
  });

  const store = createStore(reducer,
    initialState,
  compose(applyMiddleware(thunk),
    window.devToolsExtension
  ? window.devToolsExtension()
  : f => f));

  return store;
};

export default configureStore;
