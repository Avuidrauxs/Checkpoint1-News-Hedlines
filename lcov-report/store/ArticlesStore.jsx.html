<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for store/ArticlesStore.jsx</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">store</a> ArticlesStore.jsx
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>0/48</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/35</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/11</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>0/17</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54</td><td class="line-coverage quiet"><span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import { Even<span class="cstat-no" title="statement not covered" >tEmitter } from 'events';<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span>
import Dispatcher <span class="cstat-no" title="statement not covered" >from '../dispatcher';<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span>
&nbsp;
/**
 *Listens and stores data from the action according to their action type
 *
 * @class ArticleStore
 * @extends {EventEmitter}
 */
class ArticlesSt<span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" >or</span>e extends EventEmitter {<span class="cstat-no" title="statement not covered" ></span></span>
  /**
   * ArticleStore constructor
   * @return {null} - returns no value
   */
  construct<span class="fstat-no" title="function not covered" >or() {<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ></span></span></span>
    super();
<span class="cstat-no" title="statement not covered" >    this.articles = [];</span>
<span class="cstat-no" title="statement not covered" >    this.articleTitle = '';</span>
<span class="cstat-no" title="statement not covered" >    this.getAllNewsArticles = this.getAllNewsArticles.bind(this);<span class="cstat-no" title="statement not covered" ></span></span>
  }
&nbsp;
/**
 * this method just sets ths articles store with articles
 * @memberof ArticlesStore
 * @return {array} - returns an array of articles
 */<span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ></span></span>
  getAllNewsArticles() {
    re<span class="cstat-no" title="statement not covered" >turn { articles: this.articles, articleSource: this.articleTitle };</span>
  }
  /**
   * This function listens for payLoad from the action and stores them
   * according to their action type.
   * @param {object} action
   *@return {null} - returns no value
   * @memberof ArticlesStore
   */<span class="fstat-no" title="function not covered" ></span>
  handleActions(action) {
    sw<span class="cstat-no" title="statement not covered" >itch (action.type) {</span>
      case 'FETCH_ALL_ARTICLES':
        th<span class="cstat-no" title="statement not covered" >is.articles = action.articles.articleList;</span>
        th<span class="cstat-no" title="statement not covered" >is.articleTitle = action.articles.articleSource;</span>
        th<span class="cstat-no" title="statement not covered" >is.emit('change');</span>
        br<span class="cstat-no" title="statement not covered" >eak;</span>
      default:
    }
  }<span class="cstat-no" title="statement not covered" ></span>
&nbsp;
}
&nbsp;
const articlesStore <span class="cstat-no" title="statement not covered" >= new ArticlesStore(</span>);
<span class="cstat-no" title="statement not covered" >Dispatcher.register(articlesStore.handleActions.bind(articlesStore));<span class="cstat-no" title="statement not covered" ></span></span>
&nbsp;
export default articlesStore;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Mon May 22 2017 23:20:36 GMT+0100 (WAT)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
</body>
</html>
