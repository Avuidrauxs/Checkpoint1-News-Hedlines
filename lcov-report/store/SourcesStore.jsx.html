<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for store/SourcesStore.jsx</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">store</a> SourcesStore.jsx
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">50% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>5/10</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">33.33% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">50% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>5/10</span>
      </div>
    </div>
  </div>
  <div class='status-line medium'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import { EventEmitter } from 'events';
import Dispatcher from '../dispatcher';
&nbsp;
/**
 *Listens and stores data from the action according to their action type
 *
 * @class SourcesStore
 * @extends {EventEmitter}
 */
class SourcesStore extends EventEmitter {
  /**
   * SourcesStore constructor
   * @return {null} - returns no value
   */
  constructor() {
    super();
    this.sources = [];
    this.getAllNewsSources = this.getAllNewsSources.bind(this);
  }
  /**
   * this method just sets ths sources store with sources
   * @memberof SourcesStore
   * @return {array} - returns an array of news sources
   */
<span class="fstat-no" title="function not covered" >  ge</span>tAllNewsSources() {
<span class="cstat-no" title="statement not covered" >    return this.sources;</span>
  }
  /**
   * This function listens for payLoad from the action and stores them
   * according to their action type.
   * @param {object} action
   *@return {null} - returns no value
   * @memberof SourcesStore
   */
<span class="fstat-no" title="function not covered" >  ha</span>ndleActions(action) {
<span class="cstat-no" title="statement not covered" >    switch (action.type) {</span>
      case 'FETCH_ALL_NEWS':
<span class="cstat-no" title="statement not covered" >        this.sources = action.sources;</span>
<span class="cstat-no" title="statement not covered" >        this.emit('change');</span>
<span class="cstat-no" title="statement not covered" >        break;</span>
      default:
    }
  }
&nbsp;
}
&nbsp;
const sourcesStore = new SourcesStore();
Dispatcher.register(sourcesStore.handleActions.bind(sourcesStore));
&nbsp;
export default sourcesStore;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Mon May 22 2017 23:20:36 GMT+0100 (WAT)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
</body>
</html>
