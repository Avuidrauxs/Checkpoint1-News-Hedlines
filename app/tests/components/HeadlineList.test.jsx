import React from 'react';
import ReactDOM from 'react-dom';
import TestUtils from 'react-addons-test-utils';
import expect from 'expect';
import $ from 'jQuery';
import { Provider } from 'react-redux';
import { configureStore } from '../../store/configureStore';

import ConnectedHeadlineList, { HeadlineList } from '../../components/HeadlineList';
import ConnectedHeadlineItem, { HeadlineItem } from '../../components/HeadlineItem';

describe('Headlines List', () => {
  it('should exist', () => {
    expect(HeadlineList).toExist();
  });
  it('should render one or more Headline components for each news source', () => {
    const sources = [{
      id: 1,
      name: 'CNN',
      description: 'ISIS are here finally',
      url: '#'},
      {
      id: 2,
      name: 'IGN',
      description: 'No man sky is freaking scam',
      url:'#'
    }];
    const store = configureStore({
      sources
    });

    const provider = TestUtils.renderIntoDocument(
      <Provider store={store}>
        <ConnectedHeadlineList/>
      </Provider>
    );
    const headlineList = TestUtils.scryRenderedComponentsWithType(provider, ConnectedHeadlineList)[0];
    const headlineComponents = TestUtils.scryRenderedComponentsWithType(headlineList, ConnectedHeadlineItem);

    expect(headlineComponents.length).toBe(sources.length);
  })
});
