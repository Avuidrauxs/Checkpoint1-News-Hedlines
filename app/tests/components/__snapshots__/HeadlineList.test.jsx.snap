// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headlines List should render a snapshot 1`] = `<div />`;
